.github/ISSUE_TEMPLATE/code-bug-report.yml

```diff
@@ -0,0 +1,73 @@
+name: Code Bug Report
+description: File a code bug report.
+title: "[Code Bug]: "
+labels: ["code bug", "triage"]
+assignees:
+  - e10harvey, bbean23, braden6521
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: ex. email@example.com
+    validations:
+      required: false
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version of our software are you running?
+      options:
+        - develop@5fd52b5
```

Comment:
TODO: change these to a textarea rather than a dropdown until we have released versions?